rubix-tests/src/test/robotframework/shared/bks.sh

```diff
@@ -22,6 +22,10 @@ SCRIPT_LOG_FILE=${LOG_DIR}/bks-script.log
 HADOOP_DIR=/usr/lib/hadoop2
 HADOOP_JAR_DIR=${HADOOP_DIR}/share/hadoop/tools/lib
 
+DOCKER_COMPOSE_YML=${BASE_DIR}/docker/docker-compose.yml
+CLUSTER_NODE_IPS=${BASE_DIR}/docker/cluster_node_ips
```

Comment:
Can we save this file in `/tmp/rubix/tests`? This would make it easier to link this file to the test containers, using the already-existing volumes